.travis.yml

```diff
@@ -1,6 +1,9 @@
 branches:
   only:
     - develop
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y imagemagick
```

Comment:
Any ideas around it? I really struggled to find an image processing module that didn't require something installed.